src/cascadia/TerminalApp/TabManagement.cpp

```diff
@@ -357,6 +367,20 @@ namespace winrt::TerminalApp::implementation
         CATCH_LOG();
     }
 
+    // Method Description:
+    // - Sets the specified tab as the focused tab and splits its active pane
+    // Arguments:
+    // - tab: tab to split
+    void TerminalPage::_SplitTab(TerminalTab& tab)
+    {
+        try
+        {
+            _SetFocusedTab(tab);
+            _SplitPane(tab, SplitState::Automatic, SplitType::Duplicate);

```

Comment:
Okay so this is a nit, and will probably spur a bigger discussion, but this is technically different than what alt+clicking on the new tab button does today. That opens the _default profile_, it doesn't duplicate the current profile. I think it's probably fine to make the alt+click duplicate as well.

@**team** We cool with changing the alt+click on new tab button behavior? (it can be a follow-up)

Reply:
I don't follow why alt+clicking the new tab button should duplicate the current profile. The new tab button has always been "do X with the default profile", so I don't get why we should switch that to target the current profile.

IIRC there was discussion about the new tab button completely switching over to targeting the _current profile_. If we make that transition though, I'm ok with any interaction with the new tab button then operating on the current profile. (I think Leonard wanted it to run `duplicateTab` instead of `openTab` by default).

Reply:
I suppose I'm more wondering if this is a consistency issue. Is it weird that the context menu duplicates but the button uses the default?

Reply:
I assumed people would want to duplicate their profile while splitting if they used the tab context menu. That way they have the option of either duplicating with the context menu or getting their default profile by using the new tab button.

Reply:
I can buy that